crawler.py

```diff
@@ -745,26 +876,26 @@ def add_urlscan_indicators(domain, data):
         {
             "indicator_type": "2-techstack",
             "indicator_content": [tech["app"] for tech in detected_tech],
-            "domain_name": domain
+            "domain_name": domain,
         }
     )
     return urlscan_indicators
 
 
 # Send a GET request to the specified URL, ignoring bad SSL certificates
-def get_endpoints(url,endpoints):
-#     for endpoint in endpoints:
-#         response = requests.get(f"{url}/{endpoint}", verify=False)
-#         if response.status_code == 200:
-#             return response.text
-    return None
+def get_endpoints(url, endpoints):
```

Comment:
note 1: this was commented out. Was that on purpose?
note 2: this is only returning the result for the first endpoint. Is that what's intended?

Reply:
this function hits websites with basically a DDOS, let's relook at this (commented back out).